media-api/test/lib/ImageResponseTest.scala

```diff
@@ -1,11 +1,15 @@
 package lib
 
+import com.gu.mediaservice.model._
 import com.gu.mediaservice.model.usage.{PendingUsageStatus, PrintUsage, Usage}
-import com.gu.mediaservice.model.{Bounds, Crop, CropSpec}
+import lib.elasticsearch.{ElasticSearchTestBase, SourceWrapper}
 import org.joda.time.DateTime.now
+import org.scalatest.Inside.inside
 import org.scalatest.{FunSpec, Matchers}
+import play.api.libs.json._
+
+class ImageResponseTest extends FunSpec with Matchers with ElasticSearchTestBase {
```

Comment:
Unless I've misunderstood this test no longer uses the ElasticSearchTestBase?

Reply:
Refactored `ImageResponseTest` and `ElasticSearchTestBase` to resolve this.